console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/configuration/DeviceConfigPanel.java

```diff
@@ -433,6 +433,15 @@ public void handleEvent(BaseEvent be) {
         field.setAllowBlank(true);
         field.setFieldLabel(param.getName());
         field.addPlugin(dirtyPlugin);
+        if (param.getId().equals("WireGraph")) {
```

Comment:
DeviceConfigPanel is supposed to dynamically read parameters of any kura deployment.
In this code we explicitly reference parameters by their name and act accordingly. This means that kapua will have to be recompiled when new package is deployed with a field that is not in this if sequence.

Reply:
Yes, the idea is to have a set of well known fields that have a greater limit of characters. This is to fix the fact that Kura seems not providing those max values.

Does it make sense?

Reply:
You suggest that this could be a viable solution to our problem. So that one of these fields is WIreGraph constant?

Reply:
Yes. A bit dirty as a solution but it works for now, isn't it?